LibreHardwareMonitorLib/Hardware/Battery/BatteryGroup.cs

```diff
@@ -91,7 +91,7 @@ public unsafe BatteryGroup(ISettings settings)
                                                                  IntPtr.Zero))
                                     {
                                         // Only batteries count.
-                                        if (bi.Capabilities == Kernel32.BatteryCapabilities.BATTERY_SYSTEM_BATTERY)
+                                        if (0 != (bi.Capabilities & Kernel32.BatteryCapabilities.BATTERY_SYSTEM_BATTERY))
```

Comment:
Can you move the !- 0 to the end? :)

Reply:
@PhyxionNL turns out Kernel export definition had Flags annotation, used appropriate syntax instead, thanks!